flink-state-backends/flink-statebackend-changelog/src/test/java/org/apache/flink/state/changelog/ChangelogDelegateHashMapInMemoryTest.java

```diff
@@ -22,17 +22,12 @@
 import org.apache.flink.runtime.state.TestTaskStateManager;
 import org.apache.flink.runtime.state.changelog.inmemory.InMemoryStateChangelogStorage;
 
-import org.junit.Rule;
-import org.junit.rules.TemporaryFolder;
-
 /**
  * Tests for {@link ChangelogStateBackend} using {@link InMemoryStateChangelogStorage} and
  * delegating {@link HashMapStateBackendTest}.
  */
 public class ChangelogDelegateHashMapInMemoryTest extends ChangelogDelegateHashMapTest {
```

Comment:
```suggestion
class ChangelogDelegateHashMapInMemoryTest extends ChangelogDelegateHashMapTest {
```
could be package private